e2e/cypress/integration/system_console/about/edition_and_license_spec.js

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @system_console
+import * as TIMEOUTS from '../../../fixtures/timeouts';
+import {
+    promoteToChannelOrTeamAdmin,
+} from '../../enterprise/system_console/channel_moderation/helpers.js';
+
+// # Goes to the System Scheme page as System Admin
+const goToAdminConsole = () => {
+    cy.apiAdminLogin();
+    cy.visit('/admin_console');
+};
+
+describe('System console', () => {
+    before(() => {
+        cy.apiRequireLicense();
+    });
+
```

Comment:
Hi @hahmadia 
We still need to upload license in the after block of this test. If not, all the subsequent tests related to Enterprise license will fail.